src/doc/rustc/src/platform-support/aarch64-apple-darwin.md

```diff
@@ -0,0 +1,41 @@
+# `aarch64-apple-darwin`
+
+**Tier: 2 with host tools**
+
+64-bit Arm-based Apple devices running macOS: Macs with M1-family or later Apple Silicon CPUs.
+
+## Target maintainers
+
+- ???
+
+## Requirements
+
+This target supports host tools and cross-compilation. It provides the full standard library including std and alloc.
+
+Binaries built for this target expect a Apple Silicon Mac on macOS 11.0 Big Sur+.
+
+### Format
+
+The default binary format is Mach-O, the executable format used on Apple's platforms.
+
+## Building the target
+
+Just add it to the `target` with:
+```sh
+rustup target add aarch64-apple-darwin
```

Comment:
I think this section is supposed to be for building `rustc` such that it includes the target, not just for building code that uses the target.  Although looking at the other documents it's kind of inconsistently used?

Reply:
The command for building the target std is always the same, so I think this should be fine.